frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -27,8 +27,7 @@ use frame_support::{
 };
 use frame_system::RawOrigin;
 use rand::{prelude::SliceRandom, rngs::SmallRng, SeedableRng};
-use sp_arithmetic::{per_things::Percent, traits::One};
-use sp_runtime::InnerOf;
+use sp_arithmetic::{per_things::Percent, traits::One, InnerOf};
```

Comment:
```suggestion
use sp_arithmetic::{Percent, traits::One, InnerOf};
```

All types should be re-exported from the root crate, all traits must be in `traits`.